orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/MonitorKatoTask.groovy

```diff
@@ -97,7 +105,7 @@ class MonitorKatoTask implements RetryableTask {
   }
 
   // This is crappy logic
-  // TODO clean this up in Kato
+  // TODO clean this up in Kato (deprecate in favor of getDeployedNames)
```

Comment:
Since this is in a comment, why not just make it a javadoc and use `@deprecated` (and indicate what to use instead, as you have already done here)?